src/Avalonia.Visuals/Media/Fonts/FontKey.cs

```diff
@@ -4,20 +4,20 @@ namespace Avalonia.Media.Fonts
 {
     public readonly struct FontKey : IEquatable<FontKey>
     {
-        public readonly FontFamily FontFamily;
+        public readonly string FamilyName;
```

Comment:
Only just noticed this, but is it a good idea to expose fields on a public API? Should these be properties?

Reply:
Hmm. Good catch. Will make these fields private and expose values via properties.